src/components/media_control/media_control.js

```diff
@@ -134,7 +134,7 @@ export default class MediaControl extends UIObject {
   }
 
   enable() {
-    if (this.options.chromeless) return
+    if (this.options.chromeless) {return}
```

Comment:
This is another undesired result. We should either have no braces at all or have braces with padding.

Reply:
sure, ideally there should be nothing to fix in the code.

but i think that braces are a good thing. but with padding, i more than agree :+1: